spec/PQueue.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import PQueue from '../src/PQueue'
+
+chai.use(chaiChange)
+
+// Element priority starts with 1 being the highest priority
+
+describe( 'PQueue', () => {
+  let myPQueue
+  beforeEach( () => {
+    myPQueue = new PQueue()
+  })
+
+  it( 'exists', () => {
+    expect( PQueue ).to.be.a( 'function' )
+  })
+
+  context( '#enqueue( element, priority )', () => {
+    it( 'should add an object with element and priority to the queue', () => {
+      myPQueue.enqueue( "foo", 100 )
+      expect( myPQueue.front() ).to.eql( { element: "foo", priority: 100 } )
+    })
+    it( 'should add an element and position it by priority', () => {
```

Comment:
Please use whitespace to separate individual tests, in order to increase readability.